Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -228,9 +228,11 @@ The dashboards have been created by [@galexrt](https://github.com/galexrt). For
 
 The following Grafana dashboards are available:
 
-- [Ceph - Cluster](https://grafana.com/grafana/dashboards/2842)
-- [Ceph - OSD (Single)](https://grafana.com/grafana/dashboards/5336)
-- [Ceph - Pools](https://grafana.com/grafana/dashboards/5342)
+- [Ceph - Cluster (ID: 2842)](https://grafana.com/grafana/dashboards/2842)
+- [Ceph - OSD (Single) (ID: 5336)](https://grafana.com/grafana/dashboards/5336)
+- [Ceph - Pools (ID: 5342)](https://grafana.com/grafana/dashboards/5342)
+
+The dashboard JSON files are also available on [GitHub here `/Documentation/Storage-Configuration/Monitoring/ceph-monitoring/dashboards/`](https://github.com/rook/rook/tree/master/Documentation/Storage-Configuration/Monitoring/ceph-monitoring/dashboards/).
```

Comment:
How about if we keep them in Github under `deploy/examples/monitoring/grafana`, rather than the documentation folder?